### PR TITLE
Add new toasts examples

### LINKED ISSUE #28201: Dark toast variant
I would like to propose the addition of a dark variant of the toast component.

### LINKED ISSUE #28202: Alternative toast variants in documentation
I would like the documentation to provide examples of alternative variants of the toast.

For example toasts without the "toast header" and the close button.
Or a toast with an action, such as an undo button.

### PR DESCRIPTION
**Pull request by mdo**
This should cover us nicely for how to build and customize toasts. This also basically includes how to do a dark mode (use `.bg-dark`) version.

- Adds a new custom content example that removes the toast header and adds a custom close icon
- Adds an example for adding content, like more buttons
- Adds docs for how to create your own color schemes

Fixes #28201 and fixes #28202.

Preview:

* <https://deploy-preview-30572--twbs-bootstrap.netlify.app/docs/5.0/components/toasts/#custom-content>
* <https://deploy-preview-30572--twbs-bootstrap.netlify.app/docs/5.0/components/toasts/#color-schemes>

### CODE REVIEW

File: site/content/docs/4.3/components/toasts.md [hunk 1]
```diff
@@ -98,6 +98,60 @@ When you have multiple toasts, we default to vertically stacking them in a reada
...
+Customize your toasts by removing sub-components, tweaking with [utilities]({{< docsref "/utilities/api" >}}), or adding your own markup. Here we've created a simpler toast by removing the default `.toast-header`, adding a custom hide icon from [Bootstrap Icons](https://icons.getbootstrap.com), and using some [flexbox utilities]({{< docsref "/utilities/flex" >}}) to adjust the layout.
+
+{{< example class="bg-light" >}}
+<div class="toast d-flex" role="alert" aria-live="assertive" aria-atomic="true">
+  <div class="toast-body">
+    Hello, world! This is a toast message.
+  </div>
+  <button type="button" class="ml-auto p-2 close" data-dismiss="toast" aria-label="Close">
+    <span aria-hidden="true">
+      <svg class="bi bi-x" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (XhmikosR) [inline site/content/docs/4.3/components/toasts.md:112]: We should use what we currently use everywhere, `&times;`

Comment (mdo) [reply site/content/docs/4.3/components/toasts.md:112]: The point of the example here is to show a different icon :).

Comment (XhmikosR) [reply site/content/docs/4.3/components/toasts.md:112]: I'm 50-50 on this. I mean, it's generally pretty straight-forward, I'm not sure it needs an example...

Comment (XhmikosR) [reply site/content/docs/4.3/components/toasts.md:112]: Also, I'd say the extra `span` is not needed; you should move `aria-hidden` to the SVG itself.

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
Just squash and merge @mdo, but it LGTM

**Pull request merged**